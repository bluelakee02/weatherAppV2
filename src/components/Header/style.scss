.header {
  grid-area: header;
  background-color: $blue-01;
  display: flex;
  justify-content: space-evenly;
  align-items: center;

  & h1 {
    background-color: $grey-02;
    color: $grey-01;
  }
}

@media screen and (max-width: $breakpoint-tablet) {
  .header {
    flex-direction: column;
  }
}
