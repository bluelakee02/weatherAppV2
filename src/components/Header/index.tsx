import style from './style.scss';
import React from 'react';

const Header = () => <header className={style.header}>
    <h1>
        Weather App
    </h1>
</header>

export default Header
