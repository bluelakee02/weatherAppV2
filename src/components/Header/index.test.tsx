import * as React from 'react';
import {shallow, render, mount} from 'enzyme';

import Header from './index';

describe('Header test', () => {
    it('Matches the snapshot', () => {
        expect(shallow(<Header/>)).toMatchSnapshot();
    });

    it('Has proper class', () => {
        const wrapper = mount(<Header/>);
        expect(wrapper.find(`.header`)).toHaveLength(1);
    });

    it('Has proper nodes', () => {
        expect(mount(<Header/>).find('h1').length).toBe(1);
    });

    it('Renders static HTML', () => {
        expect(render(<Header/>).text()).toEqual('Weather App');
    });
});
