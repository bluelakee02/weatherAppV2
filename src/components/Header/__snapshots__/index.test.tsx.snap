// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header test Matches the snapshot 1`] = `
<header
  className="header"
>
  <h1>
    Weather App
  </h1>
</header>
`;
